tests/integration_tests/test_upgrade.py

```diff
@@ -120,7 +120,6 @@ def test_cosmovisor_upgrade(custom_ethermint: Ethermint):
 
     # get proposal_id
     ev = parse_events(rsp["logs"])["submit_proposal"]
-    assert ev["proposal_type"] == "SoftwareUpgrade", rsp
     proposal_id = ev["proposal_id"]
 
     rsp = cli.gov_vote("validator", proposal_id, "yes")
```

Comment:
Added a refactor suggestion on this [PR](https://github.com/mmsqe/ethermint/pull/5) for this test because I was getting an error message
```
upgrade cannot be scheduled in the past: invalid request
```